app/utils/IndexUtils.scala

```diff
@@ -16,42 +16,42 @@
 
 package utils
 
-import reactivemongo.api.bson.BSONDocument
-import reactivemongo.api.bson.collection.BSONSerializationPack
-import reactivemongo.api.indexes.Index.Aux
-import reactivemongo.api.indexes.Index
-import reactivemongo.api.indexes.IndexType
+//import reactivemongo.api.bson.BSONDocument
+//import reactivemongo.api.bson.collection.BSONSerializationPack
+//import reactivemongo.api.indexes.Index.Aux
+//import reactivemongo.api.indexes.Index
+//import reactivemongo.api.indexes.IndexType
 
 object IndexUtils {
 
-  def index(
-    key: Seq[(String, IndexType)],
-    name: Option[String],
-    unique: Boolean = false,
-    sparse: Boolean = false,
-    background: Boolean = false,
-    options: BSONDocument = BSONDocument.empty
-  ): Aux[BSONSerializationPack.type] = Index(
-    key = key,
-    unique = unique,
-    name = name,
-    background = background,
-    sparse = sparse,
-    expireAfterSeconds = None,
-    storageEngine = None,
-    weights = None,
-    defaultLanguage = None,
-    languageOverride = None,
-    textIndexVersion = None,
-    sphereIndexVersion = None,
-    bits = None,
-    min = None,
-    max = None,
-    bucketSize = None,
-    collation = None,
-    wildcardProjection = None,
-    version = None,
-    partialFilter = None,
-    options = options
-  )
+//  def index(
```

Comment:
Accidental commented out code?

Reply:
Yep - no longer require and will be removed.